bitnami/clickhouse/Chart.yaml

```
@@ -33,4 +33,4 @@ maintainers:
 name: clickhouse
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/clickhouse
-version: 3.7.1
+version: 3.7.2
```

These changes add a new feature, according to [semver](https://semver.org/), minor version should be increased.
```suggestion
version: 3.8.0
```